backtrace-library/src/main/java/backtraceio/library/BacktraceDatabase.java

```diff
@@ -50,6 +51,7 @@ public class BacktraceDatabase implements Database {
     private boolean _enable = false;
     private Breadcrumbs breadcrumbs;
 
+    private boolean _enabledNativeIntegration = false;
```

Comment:
name can be without _

Reply:
In following the pattern that we already have there